packages/oscal-react-library/src/components/OSCALControlPart.js

```diff
@@ -117,7 +155,31 @@ export default function OSCALControlPart(props) {
       </OSCALControlRemovedPartWrapper>
     );
   }
-
+  if (partAddPosition === Position.ENDING) {
+    return (
+      <OSCALControlPartWrapper ownerState partName={props.part.name}>
+        {replacedProse}
+        {props.part.parts?.map((part) => (
+          <OSCALControlPart
+            part={part}
+            control={props.control}
+            controlId={props.controlId ?? props.control.id}
+            parameters={props.parameters}
+            implementedRequirement={props.implementedRequirement}
+            componentId={props.componentId}
+            modificationAlters={props.modificationAlters}
+            modificationSetParameters={props.modificationSetParameters}
+            key={part.id ?? part.name}
+            isEditable={props.isEditable}
+            onRestSuccess={props.onRestSuccess}
+            onRestError={props.onRestError}
+            partialRestData={props.partialRestData}
+          />
+        ))}
+        {modificationDisplay}
```

Comment:
Why is there not a `modificationDisplayEnding` in this file?

Reply:
This block of codes in under the condition that position == Position.ENDING, that is why at first, I did not use modificationDisplayEnding there. But for code clarity, I just replaced modificationDisplay by modificationDisplayEnding.
Thanks!